pkgs/development/python-modules/dogtag-pki/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchPypi, buildPythonPackage, pytest, python3, requests, six }:
+
+buildPythonPackage rec {
+  pname = "dogtag-pki";
+  version = "11.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-rQSnQPNYr5SyeNbKoFAbnGb2X/8utrfWLa8gu93hy2w=";
+  };
+
+  checkInputs = [ pytest ];
```

Comment:
```suggestion
  checkInputs = [ pytestCheckHook ];
```

Reply:
Removed the checkInputs entirely, as there are 0 tests which causes pytestCheckHook to fail.